adage/analyze/models.py

```diff
@@ -149,6 +149,7 @@ class MLModel(models.Model):
     # False (not directed).
     directed_g2g_edge = models.BooleanField(default=False)
     g2g_edge_cutoff = models.FloatField(default=0.0)
+    desc_html = models.CharField(max_length=2048, blank=True)
```

Comment:
I was a little surprised to find that Django doesn't have any sort of `HTMLField` but I guess a `CharField` can hold HTML just fine. Did you consider using a `TextField` here instead? I don't think I have a strong opinion one way or another. The main difference is that `TextField` has no pre-set limit on length.

There is great potential for abuse of unfiltered user-provided HTML on web sites, but in this case we only allow an administrator to supply a `desc_html` (via the management command) so as long as that remains true we should be safe on that front.

Reply:
I thought about it when adding the new field. But I saw some online article claiming that `CharField` is slightly faster than a `TextField` (I forgot from where though). So I eventually used the former.

Reply:
I'd guess the trade-off is that `CharField` is less efficient in storage or memory space, but the defined `max_length` makes it faster to execute.